src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,3 @@
+databaseChangeLog:
+  - include:
+      file: db/changelog/changes/08112024-create-book-table.yaml
```

Comment:
add empty line at the end of file

Reply:
There is empty line. Please be attentive